internal/view/assets/less/custom-dialog.less

```diff
@@ -153,4 +153,23 @@
 			}
 		}
 	}
-}
\ No newline at end of file
+}
+
+
+@media only screen and (max-width: 768px) {
+	.custom-dialog-overlay {
+		padding: 0px;
+	}
+
+	.custom-dialog {
+		width: 100%;
+		height: 100%;
+	}
+}
+
+@media screen and (min-width: 769px) {
```

Comment:
I think we don't need this one, it can be the dialog defaults, isnt' it?

Reply:
Can't we put those "defaults" directly in the `.custom-dialog` class and let the media query override them only for smaller screens? Maybe I'm wrong, haven't played with CSS in a while.